modules/terraform-zscc-ccvm-aws/main.tf

```diff
@@ -37,34 +37,36 @@ resource "aws_instance" "cc_vm" {
 
 
 ################################################################################
-# Create Cloud Connector Service Interface for Small CC. 
-# This interface becomes LB0 interface for Medium/Large size CCs
+# Create Cloud Connector Service Interface for "small" CC instances. 
+# This interface becomes the Load Balancer VIP interface for "medium" and 
+# "large" CC instances.
+#
+# This primary IP Address of this interface will be used for GWLB Target Group. 
 ################################################################################
 resource "aws_network_interface" "cc_vm_nic_index_1" {
   count             = local.valid_cc_create ? var.cc_count : 0
-  description       = var.cc_instance_size == "small" ? "Primary Interface for service traffic" : "CC Med/Lrg LB interface"
+  description       = var.cc_instance_size == "small" ? "cc service interface" : "cc lb vip interface"
```

Comment:
Wouldnt it be better to leave it as cc service interface in either case? After all the concept of lb and vip is an implementation detail in our case and the end user wouldn't/shouldn't be bothered about what the interface is used internally for. To them, in either case, it will be a service traffic interface, is it not?